config_defaults.yml

```diff
@@ -41,6 +41,8 @@ appveyor.yml:
       RUBY_VER: 21-x64
   test_script:
   - "bundle exec rake spec SPEC_OPTS='--format documentation'"
+MAINTAINERS.md:
+  maintainers: "Puppet Forge Modules Team <forge-modules@puppet.com>"
```

Comment:
Multiple maintainers, I would expect this as a list and the maintainers.md file should have a foreach loop. That way it's done easily.